VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.spec.ts

```diff
@@ -90,10 +92,10 @@ describe('PrivateConsultationRoomControlsComponent', () => {
     let featureFlagServiceSpy: jasmine.SpyObj<FeatureFlagService>;
     let videoControlServiceSpy: jasmine.SpyObj<VideoControlService>;
     let videoControlCacheSpy: jasmine.SpyObj<VideoControlCacheService>;
-
     beforeAll(() => {
         featureFlagServiceSpy = jasmine.createSpyObj<FeatureFlagService>('FeatureFlagService', ['getFeatureFlagByName']);
         featureFlagServiceSpy.getFeatureFlagByName.and.returnValue(of(true));
+        ldSpy.getFlag.and.returnValue(of(true));
```

Comment:
```suggestion
        launchDarklyServiceSpy.getFlag.withArgs(FEATURE_FLAGS.wowzaKillButton).and.returnValue(of(true));
```

to avoid blanket flag settings which could interfere with other feature flag calls

Reply:
Sure, it's only use of launchdDarkly in this component. But make sense, incase we had more flags here.